projects/plugins/boost/app/assets/src/js/modules/image-size-analysis/RecommendationsMeta.svelte

```diff
@@ -49,6 +51,29 @@
 		( $isaSummary?.status === ISAStatus.Queued &&
 			__( 'Give us a few minutes while we go through your images…', 'jetpack-boost' ) );
 
+	$: foundIssuesMessage =
+		$totalPagesCount === 100
```

Comment:
The `totalPagesCount` currently shows the number of pages we are aiming to scan. My thoughts here is that, if we lift the limit to a higher number, i.e. virtually unlimited, we don't want to continue showing the number of pages scanned. For now, we only know that we are rate-limiting at exactly 100. If the check was `totalPagesCount > 100` it would not be clear if there is a rate limit or not.

But I do know that this is not ideal, and I am open to suggestions. I wish we had a way to detect if we enforced a limit. May be we should update the response from wpcom?

cc: @thingalon

Reply:
I don't see why we need to detect whether the internal page limit was hit or not. Wouldn't it be better to just say the number of pages that we scan, and ensure our documentation covers what our current limits are?